engine/src/main/java/com/datatorrent/stram/plan/physical/PhysicalPlan.java

```diff
@@ -386,6 +472,38 @@ public PhysicalPlan(LogicalPlan dag, PlanContext ctx) {
     this.undeployOpers.clear();
   }
 
+  public void setAntiAffinityForContainers(LogicalPlan dag, Collection<AffinityRule> affinityRules, Map<PTOperator, PTContainer> operatorContainerMap)
```

Comment:
Would it make sense to maintain the relationship in PhysicalPlan instead of adding to each container?